README.md

```diff
@@ -13,3 +13,19 @@ npx install-peerdeps --dev @edx/typescript-config
 ```
 
 This will install the package and all of its peer dependencies.
+
+## Usage
+
+Create file in repository `tsconfig.json`, adding references to the root directory, output directory, and directories to include/exclude in TypeScript compilation.
+
+```json
+{
+	"extends": "@edx/tsconfig",
```

Comment:
Should this be `@edx/typescript-config/tsconfig.json`? The name of this NPM package when published will be `@edx/typescript-config` per its `name` in package.json.

Reply:
Good catch!